app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -6,10 +6,14 @@ import {
   useMemo,
   useEffect,
   useImperativeHandle,
+  Ref,
 } from 'react';
 import { useI18n } from '@18f/identity-react-i18n';
 import { useIfStillMounted, useDidUpdateEffect } from '@18f/identity-react-hooks';
 import { Button, FullScreen } from '@18f/identity-components';
+import type { FullScreenRefHandle } from '@18f/identity-components';
+import type { FocusTrap } from 'focus-trap';
+import type { ReactNode, MouseEvent } from 'react';
```

Comment:
Since `Ref` is a type, can we move it into the `import type` ?

```suggestion
} from 'react';
import { useI18n } from '@18f/identity-react-i18n';
import { useIfStillMounted, useDidUpdateEffect } from '@18f/identity-react-hooks';
import { Button, FullScreen } from '@18f/identity-components';
import type { FullScreenRefHandle } from '@18f/identity-components';
import type { FocusTrap } from 'focus-trap';
import type { ReactNode, MouseEvent, Ref } from 'react';
```